### ISSUE TITLE
Note from Aymeric on quality of subsistence data for polynesia, and alternative sources

### ISSUE THREAD

**Issue by kirbykat** [D-PLACE/dplace-data#163]
Aymeric says the map of % reliance on fishing that was included in the original d-place paper includes many inaccuracies. He suggests that a reliable source for tropical polynesia is the Bishop Bulletin (authors include Peter Buck, but many prominent ethnographers). There are ~150 separate monographs, which are separated into "sections" that often correspond to D-PLACE categories, so Aymeric thinks should relatively straightforward to code. He also recommends an excellent cross-cultural study of marriage systems in Polynesia as a more reliable source for the 'transactions at marriage' variable of the EA.

**Comment by SimonGreenhill** [comment]
these should be added as new datasets.

**Comment by kirbykat** [comment]
yep, I tagged it as Dataset...maybe I'll make that clearer in the title
